RELEASE_STEPS.md

```diff
@@ -0,0 +1,83 @@
+## UMF Release Steps
+
+This document contains all the steps required to make a new release of UMF.
+
+As a helper, we use in this guide these 2 variables:
+```bash
+  set $VERSION = new full version (e.g., 0.1.0-rc1) # -rc1 included just as an example
+  set $VER = new major+minor only version (e.g., 0.1)
+```
+
+**Note:**
+> Before doing the final release, it's recommended to prepare a pre-release version - a "release candidate"
+> (or "rc" in short). This requires adding, e.g., `-rc1` to the VERSION string. When all tests and checks
+> end properly, you can follow up with the final release. If any fix is required, it should be included in
+> another rc version (e.g., `-rc2`).
+
+At the moment, UMF releases are aligned with oneAPI releases. Once all changes planned for
```

Comment:
Why the alignment information is here? I think the user wouldn't know what to do with this information. Does it mean that there will be UMF versions named after oneAPI releases or sth else?

Reply:
tbh, I followed the UR writing in here:

https://github.com/oneapi-src/unified-runtime?tab=readme-ov-file#release-process

In my understanding it means only that we will release mostly for oneAPI usage and we'll align the date for the oneAPI release date

Reply:
Maybe rephrase it so that the information that UMF will be released once for each oneAPI release is included?

Reply:
Done.